server/methods/createDirectMessage.js

```diff
@@ -8,74 +8,91 @@ import { RateLimiter } from '../../app/lib';
 import { addUser } from '../../app/federation/server/functions/addUser';
 import { createRoom } from '../../app/lib/server';
 
-Meteor.methods({
-	createDirectMessage(...usernames) {
-		check(usernames, [String]);
+function createDirectMessage(usernames, excludeSelf) {
```

Comment:
you can export this function and use it elsewhere instead of calling the new Meteor method

Reply:
Sure thing, done